Country.py

```diff
@@ -0,0 +1,270 @@
+        """
+        if self.infected_pop >= 1 and other.infected_pop == 0:
+            if random.random() <= self.infected_ratio()/10:
+                other.infected_pop = 1
+
+
+background_color = (255,255,255)
+width, height = 640, 480
+
+screen = pygame.display.set_mode((width, height))
+pygame.display.set_caption('Plague Simulation')
+screen.fill(background_color)
+
+intro = True
+
+while intro:
+    """ Intro of the game gives the intro screen with
+    upgrade instructions as well as how to start the game """
+
+    for event in pygame.event.get():
+        if event.type == pygame.QUIT:
+            pygame.quit()
+            quit()
+        if event.type == pygame.KEYDOWN:
+            if event.key == pygame.K_c:
+                intro = False
+                running = True
+            if event.key == pygame.K_q:
+                pygame.quit()
+                quit()
+    """ Renders the text where it tells the user to
+    press c to start the game, and gives instructions on how to upgrade the game"""
+
+    basicfont = pygame.font.SysFont(None, 20)
+    text = basicfont.render('Welcome To A Plague Simulation! Press C to Start, Click on a country to start your infection', True, (0, 0, 0), (255, 255, 255))
+    textrect = text.get_rect()
+    textrect.centerx = screen.get_rect().centerx
+    textrect.centery = screen.get_rect().centery
+    screen.blit(text, textrect)
+    screen.blit(basicfont.render('For Upgrades, Press I to increase infection rate, K to increase kill rate, and A to increase airborne rate' , True, (0, 0, 0), (255, 255, 255)), (0, 300 ))
+
+    pygame.display.update()
+
+screen = pygame.display.set_mode((640, 480))
+
+""" Herein, three countries are defined """
+country1 = Country(200, 120, 200, radius=60)
+country2 = Country(500, 180, 200, radius=40, color=BLUE)
+country3 = Country(320, 360, 200, radius=50, color=GREEN)
+countries = [country1, country2, country3]
+country_pop_index = country1
+total_pop = 0
+
+Time = 0
+time = 0
+Upgrade_Point = 0
+infectionindex = 1
+""" This is the counter to allow you to
+click on a country and place a pathogen"""
+clock = pygame.time.Clock()
+
+upgrades = 0
+
+""" Pressing C will officially start the game running our
+game function with time"""
+
+running = True
+while running:  # forever -- until user clicks in close box
+    for event in pygame.event.get():
+        if event.type == pygame.MOUSEBUTTONDOWN:
+            for country in countries:
+                """ start of the game, clicking the first country
+                will place the first pathogen in the country """
+                if country.contains_pt(pygame.mouse.get_pos()):
+                    if infectionindex == 1:
+                        country.infected_pop = country.infected_pop + 1
+                        infectionindex = infectionindex - 1
+                    country_pop_index = country
+        """now our pathogen embarks on infection!"""
```

Comment:
Use inline comments (like this: `# this is a comment`) instead of multiline comments in this case .